templates/fastcgi/server.erb

```diff
@@ -1,3 +1,4 @@
-FastCGIExternalServer <%= @faux_path %> -idle-timeout <%= @timeout %> <%= if @flush then '-flush' end %> -host <%= @host %>
+FastCGIExternalServer <%= @faux_path %> -idle-timeout <%= @timeout %> <%= if @flush then '-flush' end %> -host <%= @host -%>
+<%- if @pass_header -%> -pass-header <%= @pass_header %><% end %>
```

Comment:
Could you actually change these two `-%>\n<%-` to just `%> <%` so they're on the same line? It seems a little confusing to have them be on two lines if you want the output to be a single line, and flush is already setting precedent for single-line code. Thanks!